app/assets/stylesheets/overrides.scss

```diff
@@ -10,12 +11,34 @@ $navactive: $orange;
 $tabdown: $cream;
 $font: $brown;
 
+.container {
+background-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 0 spaces